src/main/java/domain/lotto/LottoPrize.java

```diff
@@ -0,0 +1,60 @@
+package domain.lotto;
+
+import java.util.Arrays;
+import java.util.function.Predicate;
+
+public enum LottoPrize {
+    FIRST(6, 2_000_000_000),
+    SECOND(5, 30_000_000),
+    THIRD(5, 1_500_000),
+    FOURTH(4, 50_000),
+    FIFTH(3, 5_000),
+    MISS(0, 0);
+
+    private final long countOfMatch;
+    private final long winningMoney;
+
+    LottoPrize(long countOfMatch, long winningMoney) {
+        this.countOfMatch = countOfMatch;
+        this.winningMoney = winningMoney;
+    }
+
+    public long getCountOfMatch() {
+        return countOfMatch;
+    }
+
+    public long getWinningMoney() {
+        return winningMoney;
+    }
+
+    public static LottoPrize valueOf(int count, boolean bonusNumber) {
+        LottoPrize lottoPrize = matchLottoPrize(count);
+
+        if (isSecondPrize(bonusNumber, lottoPrize)) {
```

Comment:
bonusNumber는 int 타입이나 LottoNumber 타입으로 오인되기 쉬워보이네요. boolean 타입에 맞는 네이밍이 필요해보입니다.

```suggestion
    public static LottoPrize valueOf(int count, boolean isMatchBonusNumber) {
        LottoPrize lottoPrize = matchLottoPrize(count);

        if (isSecondPrize(isMatchBonusNumber, lottoPrize)) {
```

Reply:
앗 그렇네요.. 다시  변수명만 들으면 int타입으로 보이는것 같습니다..

전체적으로 boolean값은 리팩토링 하겠습니다.